af/shovel/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM ubuntu:16.04
 # All the actions are done within single RUN to reduce number of layers & overall fetch size.
 RUN set -ex \
     && apt-get update \
-    && apt-get install -y python2.7 liblz4-tool python-ujson python-yaml python-psycopg2 python-six \
+    && apt-get install -y python2.7 liblz4-tool python-ujson python-yaml python-psycopg2 python-six python-statsd\
```

Comment:
Nit: maybe keep a whitespace before the `\`

Reply:
Done